gpt/git/instructions.js

```diff
@@ -0,0 +1,26 @@
+
+  const formatInstructions = (
```

Comment:
The formatInstructions function is tightly coupled with the GitCommitPromptTemplate class. Consider passing the necessary dependencies as arguments to the function to improve modularity.